libs/search/src/lib/state/effects.ts

```diff
@@ -65,6 +65,15 @@ export class SearchEffects {
             : 'no abstract',
           thumbnailUrl: hit._source.overview ? hit._source.overview.url : '',
           url: `/geonetwork/srv/eng/catalog.search#/metadata/${hit._source.uuid}`,
+          category: hit._source.resourceCategoryObject
```

Comment:
You misread the purpose of this mapping.
This is a mapper between ElasticSearch index and `RecordSimple` interface. `resourceCategoryObject`, `resourceMoreObject` and `resourceOpenObject` don't exist in ES search response.

Reply:
Ok, I will remove them